lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1587,6 +1594,9 @@ void HdVP2Mesh::_HideAllDrawItems(const TfToken& reprToken) {
         if (!renderItem)
             continue;
 
+        HdVP2DrawItem::RenderItemData& drawItemData(drawItem->GetRenderItemData());
+        drawItemData._enabled = false;
```

Comment:
This was a real problem. Later when we are calling MRenderItem::enable() we check to see if the new value is the same as the old value. If we don't update drawItemData._enabled then that check thinks the MRenderItem is enabled when it is not, and we don't show the rprim correctly.